blog/2022-10-03-six-ways-how-to-contribute-to-open-source.md

```diff
@@ -0,0 +1,71 @@
+
+![Hacktoberfest issues on GitHub](/img/blog/hacktoberfest/hacktoberfest-issues.png)
+
+## 4. Translate
+
+Translating is a very valuable kind of contribution as you help to make the software accessible to a way larger user base around the world. If you are a native speaker or you’re fluent in some other language, translation is an awesome way how to contribute. Naturally, not all open-source projects are prepared for [localization](/blog/localization-basics-S01E01), so you can get in touch with the maintainers of your favorite project and offer them your help. 
+
+**Contributing to Tolgee tip:** You can translate Tolgee tool to some new language. Now it’s available in English, French, German, Spanish, and Czech (and Norwegian is in the process). You can also review existing translations and comment on them to make them perfect.  Just shoot us a [message](mailto:info@tolgee.io)!
+
+![help translate Tolgee](/img/blog/hacktoberfest/translate-tolgee.png)
+
+## 5. Actually use open-source products
+
+Just be a great user! From the user point of view, you can e.g. improve UX or report bugs. You can find new projects on GitHub or another great place where to explore open-source software alternatives is for example [Open Source Alternative To project](https://www.opensourcealternative.to).
+
+**Contributing to Tolgee tip:** Implement Tolgee on your project to make it multilingual.
+Double contribution tip: kill two birds with one stone by contributing with translations to some open-source project which uses Tolgee and then share with us your experience. You can check for example following projects looking for translation contributions:
+
+- [Affine](https://docs.affine.pro/affine/internationalization/operation-guide) 
+- [ElevenClock](https://github.com/martinet101/ElevenClock/wiki#translating-elevenclock) 
+- [Fluent Search](https://www.fluentsearch.net/posts/fluent-search-version-09910) 
+
+## 6. Share feedback
+
+To move forward with any open-source project, feedback from real users is very needed. Do you have any ideas on how to improve the project? Which feature to add? Share it in project communities and discuss it with others. 
+
+**Contributing to Tolgee tip:** suggest new features, share your feedback in our [Slack Tolgee community](https://join.slack.com/t/tolgeecommunity/shared_invite/zt-195isb5u8-_RcSRgVJfvgsPpOBIok~IQ) or on [Github Discussions](https://github.com/tolgee/tolgee-platform/discussions)
+
+## Final thoughts
+Of course, there are definitely more options on how to contribute to open-source out there. If your motivation is whatsoever from the willingness to learn something new, improve your existing skills or just support your favorite open-source project, you have multiple options on how to do it. But the important thing is that everyone willing to contribute to open-source can actually bring their skills to the community to support it. 
```

Comment:
Maybe "Whatever your motivation, ..., you have multiple options on how to do it"